compiler/src/dotty/tools/dotc/typer/Typer.scala

```diff
@@ -833,12 +833,18 @@ class Typer extends Namer
       case _: WildcardType => untpd.TypeTree()
       case _ => untpd.TypeTree(tp)
     }
+    def interpolateWildcards = new TypeMap {
+      def apply(t: Type) = t match
+        case WildcardType => newTypeVar(TypeBounds.empty)
+        case WildcardType(bounds: TypeBounds) => newTypeVar(bounds)
```

Comment:
Could there be WildcardTypes in the bounds themselves that need to be replaced recursively ?